src/docs/03-case-studies/docskit-mdx-partials.mdx

```diff
@@ -1,26 +1,29 @@
 ---
-title: Centralizing reusable content with DocsKit Partials
+title: Single sourcing reusable content with DocsKit Partials
```

Comment:
It reads as a tautology to me because "single sourcing" already implies that you are reusing content. Can you think a bit more on the title? 🤔 the keywords are single sourcing, content library, centralized, content blocks, reusing, but single sourcing and reusing are too much together.

If we want go with Occam's razor, it is "Single sourcing content with DocsKit Partials".

Reply:
Thank you! I edited it to be "Single sourcing content with DocsKit Partials".